README.md

```diff
@@ -93,10 +94,12 @@ Timeline playback is multi-threaded using the C# Job System.
 ## Variants, Models, & Payloads
 
 Access to variant selection, model details, and payload state are all accessible via
-the inspector on the game object at which these features were authored.
+the inspector on the game object at which these features were authored. Note that Payloads *are not loaded by default*, and USD files using Payloads must be reloaded after changing the Payload Policy in the Inspector to 'Load All' for them to appear in the Scene. 
```

Comment:
Very minor suggestion:
Note that Payloads *are not loaded by default*. USD files using Payloads must be reloaded after changing the Payload Policy in the Inspector to 'Load All' for them to appear in the Scene.